src/components/TvDetails/index.tsx

```diff
@@ -386,9 +386,9 @@ const TvDetails: React.FC<TvDetailsProps> = ({ tv }) => {
             </span>
           </div>
           <h1 className="text-2xl lg:text-4xl">
-            <span>{data.name}</span>
+            {data.name}{' '}
             {data.firstAirDate && (
-              <span className="ml-2 text-2xl">
+              <span className="text-2xl">
```

Comment:
Why are we not using the `ml-2` here for spacing? This seems like an odd change?

Reply:
I noticed it was different on the MovieDetails and just made it consistent.  Do you prefer `ml-2`?  I think using an actual space is better, since it would make more sense without the formatting.

Reply:
I don't agree using an actual space is better. We should avoid using spaces for layout purposes.

Reply:
The movie details page is just outdated. I fixed TvDetails but never brought the change back over.

Reply:
Ah okay, sure.  Either way, I just wanted it to be consistent 😺